src/page-modules/contact/validation/commonInputValidator.ts

```diff
@@ -139,11 +139,7 @@ export const commonInputValidator = (context: any) => {
     ),
   );
 
-  // Populate context.errorMessages
-  context.errorMessages = inputErrorMessages;
-
-  // Return false if any error
-  return Object.keys(context.errorMessages).length > 0 ? false : true;
+  return inputErrorMessages;
```

Comment:
I think you need to update the travel-guarantee form with this change?

Reply:
Yes, i will update it👍